src/modm/board/mini_f411/board.hpp

```diff
@@ -62,13 +62,12 @@ struct SystemClock {
 	enable()
 	{
 		Rcc::enableExternalCrystal();
-		Rcc::enablePll(
-			Rcc::PllSource::ExternalCrystal,
-			25,		// 25MHz / N=25 -> 1MHz
-			336,	// 1MHz * M=336 -> 336MHz
-			4,		// 336MHz / P=4 -> 84MHz = F_cpu
-			7		// 336MHz / Q=7 -> 48MHz for USB
```

Comment:
What about `.pllQ`?

The STM32F411CEx has a Q divider in its pll:

![grafik](https://user-images.githubusercontent.com/2820734/77225342-757e2f80-6b6e-11ea-9247-f9a40f1d14d5.png)

Reply:
Yeah, what about pllQ??? I couldn't find that pllQ was *ever* implemented in `:platform:rcc`. I think what happend here is that `uint32_t waitCycles = 2048` was set to 7. 😳

Reply:
Hmmm… I remember I made the call to simplify the RCC code and remove pllQ some time ago. The code is still there, it's just commented out. The reasoning was that until modm has  USB story, there's no need to worry about this.

Reply:
Ok. then we just ignore Q for now.